cfme/tests/generic_objects/test_instances.py

```diff
@@ -254,3 +254,23 @@ def test_generic_objects_tag_rest(appliance, generic_object, tags):
         generic_object.remove_tag(tag)
         tag_available = generic_object.get_tags()
         assert tag.id not in [t.id for t in tag_available]
+
+
+@pytest.mark.manual
+@pytest.mark.ignore_stream("5.10")
+def test_import_export_generic_object():
+    """
+    Bugzillas:
```

Comment:
Required: Let's the same style for referencing BZs
Should be
```
Bugzilla:
    1595259
```
See here, for example https://github.com/ManageIQ/integration_tests/pull/8675/files#diff-df310a5bec51c509260e4f0afbe8eed0R435

Reply:
Agreed, I'm going to open a PR to address this.

Reply:
Done by #8684, will probably need rebase after #8675 is merged

Reply:
@john-dupuy Thanks, can you follow the same pattern for these new tests, they are not in scope of the #8684